### DISCUSSION TITLE
Separate ingestion/management interfaces

### DISCUSSION THREAD

**Discussion by mttaggart** [Security-Onion-Solutions/securityonion#5793] [Unsupported Versions]
Hello! I tried searching through as many previous discussions as possible, but could not find an answer.

My network design is pretty standard, I think, but I'm really struggling to get this working in SO. Here's what I have:

- Management IP on subnet `192.168.99.0/24`
- 2nd IP intended for log ingestion on isolated network `10.1.99.0/24`
- Sniffing interface on the isolated network

I am accessing the web ui successfully via the primary management IP. However, I _need_ to have the log ingestion (Wazuh, for now, but any would apply) come in through the 2nd IP. I have tried updating firewall rules, looking for where Salt is setting the configs for Wazuh, but I'm just not finding what would work. Any help would be appreciated.

**Comment by dougburks** [comment]
I tested this on VMware Fusion and it appears to be working for me:
- Security Onion VM management IP on subnet `172.16.224.0/24`
- Security Onion 2nd IP for Wazuh log ingestion `172.16.91.0/24`
- separate Ubuntu VM with interface in `172.16.91.0/24`
- ran `so-allow` and chose the `w` option and allowed `172.16.91.0/24`
- installed Wazuh agent, imported key, and it connected to the manager and sent logs properly

![image](https://user-images.githubusercontent.com/1659467/155153311-95f0cd5f-8544-44b9-a8f3-d79b0f92e19d.png)

Have you tried running `so-allow` and choosing the `w` option?
https://docs.securityonion.net/en/2.3/so-allow.html

**Reply by mttaggart** [reply]
Thank you for the response! Yes, I did run `so-allow` with the `w` option for the internal collection interface. This did not work for me, nor did attempting to use any other data collection method, including Beats/OSQuery. When investigating the configs in the running containers, I found that each service was configured to only listen on the primary management interface, and I was unable to find how/where to properly change that.

Nevertheless, I will try again when I have a chance. Thank you again for looking into this.

**Reply by dougburks** [reply]
Here are a couple of other things you can check:
- make sure you are installing Wazuh agent version 3.13 to match the Wazuh server version in Security Onion. You can download the agent directly from the [SOC Downloads page](https://docs.securityonion.net/en/2.3/downloads.html).
- make sure the agent matches the server's protocol (3.13 uses UDP by default)
